sql-join/sassy-cast.sql

```diff
@@ -0,0 +1,7 @@
+select "films"."title",
+       "actors"."firstName",
+       "actors"."lastName"
+from "castMembers"
+join "films" using ("filmId")
+join "actors" using ("actorId")
+where "films"."title" = 'Jersey Sassy';
```

Comment:
The film's `"title"` shouldn't be `select`ed:

---

![image](https://user-images.githubusercontent.com/16143771/118709927-75a9a780-b7d2-11eb-8035-8b564ea9ca49.png)

Reply:
Sorry! I left that for a bit of clarity, since I'm not aware of a way to title our overall queries.

Reply:
You can always add comments: https://www.techonthenet.com/postgresql/comments.php